### PR TITLE
Use of this playground with 1.4.10

### PR DESCRIPTION
**Pull request by rachelcarmena**
This repository is used in a test from Arrow Meta so this branch will be used instead of the main branch for https://github.com/arrow-kt/arrow-meta/pull/706

It should be merged after https://github.com/arrow-kt/arrow-meta/pull/706 and then changing the use of the main branch again from Arrow Meta IDE Plugin tests.

The check will pass after merging  https://github.com/arrow-kt/arrow-meta/pull/706

### CODE REVIEW

File: src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt [hunk 1]
```diff
@@ -8,5 +8,6 @@ class RefinementTests{
     @Test
     fun `NonEmptyArray should not be empty`() {
         val x: NonEmptyArray = NonEmptyArray(emptyArray())
+        consumer.TwitterHandle("@admin")
```
Comment (danimontoya) [inline src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt]: Why creating a TwitterHandle here?

**Review by danimontoya — APPROVED** [review]
Thanks @rachelcarmena 🙌

File: src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt (hunk 1 above)
Comment (rachelcarmena) [reply src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt]: Good point! It was added by @i-walker . Please, @i-walker :pray:

Comment (danimontoya) [reply src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt]: Btw tests for refined types are in the `compiler-plugin` module:
`arrow.meta.plugins.refinement.RefinementTests`

Comment (rachelcarmena) [reply src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt]: Right, they are here as well for the Gradle test in Arrow Meta IDE Plugin

Comment (danimontoya) [reply src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt]: I removed it temporarily.. if in the end was wanted, we can create a separate test for it 👍

**Pull request merged**